lib/tab/tab-content.js

```diff
@@ -0,0 +1,62 @@
+import React, { Component } from 'react';
+import { polyfill } from 'react-lifecycles-compat';
```

Comment:
We don't need this polyfill, we are using the latest version of React.

Reply:
ok. removed. thanks